types/testingutils/runner.go

```diff
@@ -156,6 +156,7 @@ var ConstructBaseRunnerWithShareMap = func(role types.RunnerRole, shareMap map[p
 			km,
 			opSigner,
 			valCheck,
+			&ssv.Config{Domain: TestingSSVDomainType},
```

Comment:
I think we could use a variable here since it seems that all other configs are equal, right? *Similar as you did in the next function
Or is there a purpose for it to be created for every runner?